app/retail/templates/emails/start_work_applicant_about_to_expire.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize %}
+{% load static %}
+
+{% block content %}
+
+<h1>{% trans "24 Hrs to Approve" %}</h1>
+<div style="text-align: left! important">
+    <p>
```

Comment:
Can we switch this to two-space indentation for all of the new html templates throughout?

Reply:
let me see how to auto lint this

Reply:
fd6d0c388a34d7ccd1f504db065eda7a5156ac08